new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItJrfDomainInPV.java

```diff
@@ -41,7 +79,10 @@
   private static final String RCUSCHEMAPREFIX = "jrfdomainpv";
   private static final String ORACLEDBURLPREFIX = "oracledb.";
   private static final String ORACLEDBSUFFIX = ".svc.cluster.local:1521/devpdb.k8s";
-
+  private static final String RCUSYSUSERNAME = "sys";
+  private static final String RCUSYSPASSWORD = "Oradoc_db1";
+  private static final String RCUSCHEMAUSERNAME = "myrcuuser";
+  private static final String RCUSCHEMAPASSWORD = "Oradoc_db1";
 
   private static String dbUrl = null;
   private static int dbPort = getNextFreePort(30000, 32767);
```

Comment:
Do not evaluate and assign this until you are ready to use it.

Reply:
Fixed